doc/quickstart.rst

```diff
@@ -0,0 +1,488 @@
+
+   micro node.conf
+
+The default is usually ``wemos_d1_mini``. Change it if you're using a different board:
+
+.. code-block:: bash
+
+   board="wemos_d1_mini"
+   # or: board="nodemcu"
+   # or: board="esp32dev"
+   # or: board="esp32minikit"
+
+Defining Devices in setup.cpp
+-----------------------------
+
+Edit the ``setup.cpp`` file to define your devices:
+
+.. code-block:: bash
+
+   micro setup.cpp
+
+For a simple example with a button and LED, add:
+
+.. code-block:: cpp
+
+   // Button on pin D3
+   input(button1, D3, "released", "pressed").with_debounce(5);
+   
+   // Onboard LED (note: inverted for most ESP boards)
+   output(blue_led, ONBOARDLED).inverted();
+
+Save and close the file.
+
+Flashing Your First Node
+========================
+
+Connect your microcontroller to your computer via USB.
+
+Platform-Specific Serial Flashing
+---------------------------------
+
+**On Linux:**
+
+1. Find your USB device:
+
+   .. code-block:: bash
+
+      ls /dev/ttyUSB*
+      # or: ls /dev/ttyACM*
+
+2. You may need to add yourself to the ``dialout`` group (one-time setup):
+
+   .. code-block:: bash
+
+      sudo usermod -a -G dialout $USER
+      # Then log out and log back in
+
+3. Flash the node:
+
+   .. code-block:: bash
+
+      deploy serial
+
+**On Windows (WSL2):**
+
+1. Install usbipd-win (see `Installation Guide <installation.rst#usb-flashing-in-wsl2-with-usbipd-win>`_)
+
+2. In Windows PowerShell (as Administrator):
+
+   .. code-block:: powershell
+
+      usbipd list
+      usbipd attach --busid X-Y  # Use the busid from the list
+
+3. In WSL, the device should appear as ``/dev/ttyUSB0``
+
+4. Flash the node:
+
+   .. code-block:: bash
+
+      deploy serial
+
+**On macOS:**
+
+1. Find your USB device:
+
+   .. code-block:: bash
+
+      ls /dev/cu.*
+      # Usually /dev/cu.usbserial-* or /dev/cu.SLAB_USBtoUART
+
+2. Flash the node:
+
+   .. code-block:: bash
+
+      deploy serial
+
+**Alternative: Network Flashing via Router**
+
+If you have an OpenWRT router with USB ports and ser2net configured:
+
+.. code-block:: bash
+
+   deploy serial rfc2217://192.168.8.1:5000
+
+(Replace with your router's IP and configured port)
+
+Monitoring Your Node
+====================
+
+After flashing, you can monitor your node in two ways:
+
+Serial Console
+--------------
+
+In a new terminal:
+
+.. code-block:: bash
+
+   iot
+   cd ~/iot-systems/my-home/living-room-lamp
+   console_serial
+
+You'll see debug output and device state changes.
+
+MQTT Messages
+-------------
+
+In another terminal:
+
+.. code-block:: bash
+
+   iot
+   mqtt_listen
+
+Now press the button on your microcontroller. You should see messages like:
+
+.. code-block::
+
+   living-room-lamp/button1 released
+   living-room-lamp/button1 pressed
+
+Congratulations! Your first node is working!
+
+Over-the-Air Updates
+====================
+
+After the initial serial flash, you can update your node over WiFi.
+
+1. Make changes to ``setup.cpp``
```

Comment:
suggest changing debounce rate from 5 to 10 (maybe explain a bit the reason for debouncing here)